src/main/java/net/dv8tion/jda/api/JDA.java

```diff
@@ -2374,6 +2375,18 @@ default Emote getEmoteById(long id)
      * @see #shutdown()
      */
     void shutdownNow();
+    
+    /**
+     * Blocks until this JDA instance fully shuts down. <b>This method does not shut down JDA.</b>
+     * 
+     * <p>If you want to run code after shutting down async, use the {@link ShutdownEvent} in preference to this method.
```

Comment:
I don't think you should add an import just for a javadoc link, try doing:
```java
{@link net.dv8tion.jda.api.events.ShutdownEvent}
```

Reply:
```java
{@link net.dv8tion.jda.api.events.ShutdownEvent ShutdownEvent}
```
Is the correct way